### ISSUE TITLE
Saving for vehicles inside other vehicles

### ISSUE THREAD

**Issue by nward** [A3Wasteland/ArmA3_Wasteland.Altis#281]
If a vehicle is loaded inside another vehicle (ie. a kart or quad inside an offroad), when the server restarts the vehicle is lost.

This is not unreasonable, but it would be nice if:
- saveable vehicles, stored inside other saveable vehicles, were saved and stayed inside
- saveable vehicles, stored inside non-saveable vehicles, were saved at that position so they would stay there through a restart (as a bit of a stretch)

**Comment by Beefaloaf** [comment]
The 2nd request is far fetched for sure...  It would require a constant update of the locations of all stored objects, so that in the case their "parent" storage object vanishes, the "child" objects could drop reliance on the "parent". Nasty!  (My logic may be way off here btw)
At any rate, here's #279.

**Comment by nward** [comment]
Right, I didn't realise there was an issue already! I'll make some comments there..
